lib/gpu_memory_service/integrations/vllm/patches.py

```
@@ -132,6 +132,33 @@ def patched_register_kv_caches(self, kv_caches):
+            kv_mgr = get_gms_client_memory_manager("kv_cache")
+            if kv_mgr is not None and kv_mgr.has_plan_a_scratch:
+                self._plan_a_pending_kv_caches = kv_caches
+                logger.info(
+                    "[GMS Patch] Deferring NIXL KV cache registration "
+                    "(Plan A scratch; stashed %d layers for wake replay)",
+                    len(kv_caches),
+                )
+                return
+        except Exception:
+            # If we can't tell, fall through — the original behavior is safe.
+            pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)